index.js

```diff
@@ -413,6 +415,10 @@ function testGen(swagger, config) {
   source = read(join(__dirname, '/templates/environment.handlebars'), 'utf8');
   environment = handlebars.compile(source, {noEscape: true});
 
+  if (config.maxLen && config.maxLen >= 80) {
```

Comment:
I don't know why I did that, maybe got lost at that time, will change............